plat/arm/css/drivers/scp/css_pm_scmi.c

```diff
@@ -330,7 +321,7 @@ static int scmi_ap_core_init(scmi_channel_t *ch)
 
 void __init plat_arm_pwrc_setup(void)
 {
-	channel.info = &plat_css_scmi_plat_info;
+	channel.info = plat_css_get_scmi_info();
```

Comment:
Suggest reordering these so that the call to plat_css_get_scmi_info() is after channel.lock =, to preserve declarations before code (it goes from being an assignment to a function call and assignment)